src/codeu/chat/DerbyStore.java

```diff
@@ -32,6 +35,10 @@
 	private final String messageTableName = "message";
```

Comment:
It would be cool to see a diagram showing the relationships between these tables in the design doc if you get a chance.